db/migrate/20160324174218_create_container_deployment_nodes.rb

```diff
@@ -0,0 +1,17 @@
+class CreateContainerDeploymentNodes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_nodes do |t|
+      t.string :address
+      t.string :name
+      t.text :labels
+      t.belongs_to :container_deployment, :type => :bigint
+      t.belongs_to :vm,                   :type => :bigint
+      t.text :docker_storage_devices, :array => true, :default => []
+      t.string :docker_storage_data_size
+      t.text :customize
+      t.boolean :deploy_metrics, :default => true
+      t.boolean :deploy_registry, :default => true
+      t.timestamps
```

Comment:
Minor, but can you align the symbols here and in the other migrations, for readability?